pkg/gpu/stats.go

```diff
@@ -78,25 +80,45 @@ func (g *statsGenerator) getStats(nowKtime int64) *model.GPUStats {
 		}
 	}
 
-	normFactors := g.getNormalizationFactors()
-
-	stats := &model.GPUStats{
-		Metrics: make([]model.StatsTuple, 0, len(g.aggregators)),
-	}
+	// Compute unnormalized stats first for each aggregator
+	unnormalizedStats := make([]model.StatsTuple, 0, len(g.aggregators))
```

Comment:
let's call it `rawStats`
(you can keep the comment mentioning those stats are not normalized yet)